### PR TITLE
docs: add ruby docs

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

provide ruby sdk docs

### CODE REVIEW

**Comment by gao-sun** [comment]
per offline discussion, i'll merge this first to unblock our new users. the team will leave comments later or in the Slack channel.

File: docs/quick-starts/framework/ruby/README.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+slug: /quick-starts/ruby
```
Comment (charIeszhao) [inline docs/quick-starts/framework/ruby/README.mdx:2]: Do we have ruby logo?

Comment (gao-sun) [reply docs/quick-starts/framework/ruby/README.mdx:2]: no, i'll add it later

Comment (gao-sun) [reply docs/quick-starts/framework/ruby/README.mdx:2]: https://github.com/logto-io/docs/pull/696

**Pull request merged**